### PR TITLE
fix(client): move sessions to bottom

### PR DESCRIPTION
**Pull request by the-j0k3r**
- [x] The commit message follows the rest of the commits and passes the post-commit test
- [ ] If you've introduced a new feature, you also wrote tests for it and they all pass

## New behaviour:  

@DecentM I just copied and moved code to bottom please check its OK :)

This change improves usability when having sessions enabled the main account management is accessible and near the submit changes button.

Addresses 1 issue in https://github.com/OpenUserCSS/openusercss.org/issues/101

### This is suppose to move session from top as is.
screenshot-2018-6-16 openusercss

### To this
screenshot-2018-6-16-openusercss

### CODE REVIEW

**Comment by DecentM** [comment]
Thanks for the PR!  
I don't think it would be a good idea to move sessions to a more difficult to notice position, since it's an important security feature in my opinion.  

Why do you think it's important to move that small box down? Is there an issue with it I'm missing?

**Comment by the-j0k3r** [comment]
Yes, usability, when you turn on the sessions the other boxes are below, and far from the top button.

Ive opened a ticket about it, and this fixes part of it.

**Comment by the-j0k3r** [comment]
>I don't think it would be a good idea to move sessions to a more difficult to notice position, since it's an important security feature in my opinion.

If you move it down, you can still see it and have it turned onalways. So really this is a moot point.

TLDR
Actually, I really started thinking about this, currently there's 16 sessions being logged (on my side) this is longer than long and the email/password boxes are shoved into scrolling land or have to disable that in order to see this and then enable that again...Adding extra steps in order to edit any relevant account details unrelated is a usability issue..

Sessions maybe a security thing and there is no reason why it cant move two boxes down considering how intrusive it is and you can still see it without scrolling, while the same cant be said for email/pass boxes.

Really you can still see it if its moved, and if interested then you can scroll down to see that info to ad nauseam.

**Comment by Martii** [comment]
> one picture speaks 1000 words

Needs 2000 please :wink: ... What does this look like on a narrow viewport? Portables are starting to get attention as of late with the developer communities.

**Comment by the-j0k3r** [comment]
>What does this look like on a narrow viewport? Portables are starting to get attention as of late with the developer communities.

Emulate one with your browser and find out.

**Comment by Martii** [comment]
Emulation doesn't always reveal browser differences depending on how it's done on the host site. I've encountered some differences between the real thing and emulation. To sell your point on moving the sessions list lower, if OUCSS has the right panel dynamically adjusted and puts those below, you end up with the same issue all over again for portables.

**Comment by DecentM** [comment]
Wait, it's supposed to be closed by default, is it open for you when you load the account page?  

There are no checks for the browser other than for putting its name in the session list. In theory, it should behave just like on a real device if you resize the window. That said, mobile browsers can render content differently, even if that's off-spec.

**Comment by Martii** [comment]
@DecentM 
It's closed when I signed up from desktop. Think I may have visited it again at a later point and it was still closed... not available to retest again atm.

**Comment by the-j0k3r** [comment]
> Wait, it's supposed to be closed by default, is it open for you when you load the account page?

Its not the point.

You dont want to improve usability close this.

**Comment by DecentM** [comment]
I understand that from your point of view, my feedback can be seen as dragging this out. It's inevitable that everyone will have different ideas and I try to incorporate them as I can. There are no deadlines in this project, since it's my hobby. Thus, we can afford to churn over multiple issues over longer periods of time, and this will result in a better overall product that we can accept and be happy with.  

That said, I came up with an idea of how the UX on this page can be improved without sacrificing scroll time (keep in mind that additional boxes will become available later on as the number of options increase). If the page is split up like this, it will improve the aspect you're going for and will keep accessing security related features easy as well. What do you think?  

account-mock-bg  

To the left is a navigation menu, and to the right would be the cards that are currently swamped together in columns, but in a simpler way.

**Comment by the-j0k3r** [comment]
I dont mind a healthy discussion, :)

Your design using a sidebar navigation is a way to go, but essentially you are creating new pages where everything has its own place this has upsides and downsides.

An upside is nothing gets in the way of something else like the case here,

This design is bad with a sidebar when you consider mobile users you just dont have the horizontal space in some devices even when the mobile device is in horizontal position and not up right.

That said, in devices with 720p/1080p screens will be slightly better (if you are considering mobile good support) from everyone I know with good devices turns off the orientation, I dont turn it off and have better experiences overall. So you may want to use a stack for mobile devices instead of sidebar.

Desktop side its dealers choice, I like ver

**Comment by DecentM** [comment]
Merged into https://github.com/OpenUserCSS/openusercss.org/issues/124
